### DISCUSSION TITLE
Connect prisma cloud with Bitbucket

### DISCUSSION THREAD

**Discussion by IreneUelzPay** [prisma/prisma#12657] [Q&A]
Hello, 

I would like to know if its posible to connect cloud prisma with a bitbucket repository.

Thank you,
Irene

**Comment by nurul3101** [comment]
Hey @IreneUelzPay 👋 

As of now, only GitHub is supported by Prisma Cloud. 
We do plan to add support for GitLab and Bitbucket in future, but at the moment you would need to create a temporary repository in GitHub to use it with Prisma Cloud.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
